### ISSUE TITLE
Tree: on focus announce up / down arrows are used for navigation

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52658]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Extensions
3. Type something in the Search textbox, and press tab key

**Actual:**
NVDA does not read instructions for navigating inside the treeview.

**Expected:**
When the focus is on treeview, NVDA should read use up/down arrow keys to navigate inside the treeview.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User might face problem in navigating inside the treeview if screenreader would announce the instructions.

Attachment for Reference: 
[A11y VSCode Usability Extensions missing instructions.pptx](https://github.com/Microsoft/vscode/files/2126539/A11y.VSCode.Usability.Extensions.missing.instructions.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Disagree. We use the same navigation in all our trees across the workbench.
Though assigning to @joaomoreno to consider announcing that up/down are used for navigation whenever tree gets focus. IMHO this is superflous

**Comment by isidorn** [comment]
@joaomoreno I see you have tackled this only for the extensions view? Should we do it for all other views as well?

**Comment by joaomoreno** [comment]
Pushed one more commit.
